JavaDirectory/src/main/java/com/bug/free/invention/api/classes/Job.java

```diff
@@ -0,0 +1,109 @@
+package com.bug.free.invention.api.classes;
+
+public class Job {
+    int jobID;
+    String jobTitle;
+    String jobSpec;
+    int bandId;
+    int capabilityId;
+    String capabilityName;
+    String bandName;
+    String familyName;
+    int bandLevel;
```

Comment:
Band and Capability look like different "things" that should be in their own classes
```suggestion
    Band band;
    Capability capability;
    String familyName;
```

Reply:
This would require the models of other use cases to be done, and would be easier left until more merges have come in.